client/signup/steps/site-title/index.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+
+import SignupSiteTitle from 'components/signup-site-title';
+import SiteTitleExample from 'components/site-title-example';
+
+import { setSiteTitle } from 'state/signup/steps/site-title/actions';
+
+const SiteTitleStep = React.createClass( {
+	displayName: 'Site Title',
+
+	contextTypes: {
+		store: React.PropTypes.object,
+	},
```

Comment:
No, removed them where they were existing (`site-title` step and `signup-site-title` component).